### PR TITLE
Added 'Call For Volunteers' announcement

### PR DESCRIPTION
**Pull request by vrurg**
And related paper to keep the list of projects looking for help.

### CODE REVIEW

File: announcements/20211002.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+10 October 2021 Call For Raku Infrastructure Volunteers
```
Comment (japhb) [inline announcements/20211002.md:1]: Wrong date?

File: announcements/20211002.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Raku Steering Council is aware of this problem and plan to work out any
+necessary technical and administrative solutions to assist in resolving it. But
+none of them would be viable without volunteers joining the service maintainers
+in their efforts. For this reason RSC opens a recruiting program aimed at
+helping any infrastructure project maintainer in finding volunteers willing to
+share their duties.
+
+More information about the program is provided in [corresponding
+paper](../papers/Infrastructure_Project_Contact_List.md), where the maintainers
```
Comment (japhb) [inline announcements/20211002.md:18]: s/paper/document/

File: papers/Infrastructure_Project_Contact_List.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# RAKU INFRASTRUCTURE PROJECT LIST
+
+This paper is intended to hold a list of Raku infrastructures projects where
```
Comment (japhb) [inline papers/Infrastructure_Project_Contact_List.md:3]: s:g/paper/document/

File: papers/Infrastructure_Project_Contact_List.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+addition to this paper. A new project entry must be added under the _PROJECT
+LIST_ section and contain the following elements:
+
+1. Project title. It must be a second-level markdown heading like: _## IRC Bot
+   For Something_
+2. Brief project description.
+3. Perhaps a few words on what kind of help is required.
+4. Maintainer contact information. This could be anything that would allow a
+   volunteer to get in touch with you: an email (though don't forget to take
+   necessary measures to complicate spammers life), and IRC handle, a social
```
Comment (japhb) [inline papers/Infrastructure_Project_Contact_List.md:18]: "an" IRC handle

File: announcements/20211002.md (hunk 1 above)
Comment (vrurg) [reply announcements/20211002.md:1]: Sure it is! It was supposed to be month, initially...

File: papers/Infrastructure_Project_Contact_List.md (hunk 4 above)
Comment (vrurg) [reply papers/Infrastructure_Project_Contact_List.md]: Thanks, both corrections addressed in edd54d0

File: announcements/20211002.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+02 October 2021 Call For Raku Infrastructure Volunteers
+-------------------------------------------------------
+
+As Raku growing as a project its infrastructure is growing too. We get more and
```
Comment (JJ) [inline announcements/20211002.md:4]: Nit: Raku *is* growing or "As Raku grows"

File: announcements/20211002.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+more services ranging from web sites, like [the home page](https://raku.org), to
+IRC bots, to zef module ecosystem, etc. On many occasions these services are
+being ran and managed by single person making them fail-prone in cases when the
```
Comment (JJ) [inline announcements/20211002.md:7]: by *a* single person. Also, poor old them, don't say they are fail-prone, maybe rephrase it more diplomatically saying it has a low *bus factor*, or say it might cause systemic problems "... when the person, for any reason..."

File: announcements/20211002.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+person, for any reason, is unable to take necessary actions to fix things up.
+
+Raku Steering Council is aware of this problem and plan to work out any
```
Comment (JJ) [inline announcements/20211002.md:10]: *The* RSC

Comment (JJ) [reply announcements/20211002.md:10]: *plans* to work, or "We, as the RSC, plan"

**Review by JJ — CHANGES_REQUESTED** [review]
Some nits have been picked. Thanks for the effort.

File: announcements/20211002.md (hunk 6 above)
Comment (tbrowder) [reply announcements/20211002.md:7]: "being ran" should be "being run"

Comment (vrurg) [reply announcements/20211002.md]: > don't say they are fail-prone

I chose _'higher-risk zone'_ definition. Not sure if _"a"_ is needed before it, though.

**Comment by vrurg** [comment]
Thanks, @JJ and @tbrowder!

**Comment by vrurg** [comment]
I hope to get it merged today to allow @lizmat to publish it in the next Weekly News issue.

**Comment by lizmat** [comment]
Note that this week's RWN will be published on Tue rather than on Mon, due to circumstances :-)

**Comment by vrurg** [comment]
> Note that this week's RWN will be published on Tue

It's not often when I say "glad to hear" on such occasion, but now it is exactly the case! :)

**Review by JJ — APPROVED** [review]
Thanks!
